code/modules/ruins/lavalandruin_code/techcult.dm

```diff
@@ -157,6 +157,7 @@
 	l_pocket = /obj/item/kitchen/knife/combat/survival
 	r_pocket = /obj/item/tank/internals/emergency_oxygen/double
 	id = /obj/item/card/id/away/techcult
+	backpack_contents = list(/obj/item/mechandrites)
```

Comment:
There's no need to give it to every single one of them.
I'd just give it to the leader.

Reply:
Feels eh to not have the actual tech-priests (even if only their leader) be able to use them when miners can have a random chance to get them, themselves.

Unless we're going with the Leader is the actual tech-priest and the rest are just sort of adepts